src/components/WorldACLPage/WorldACLPage.tsx

```diff
@@ -201,13 +201,12 @@ export default function WorldACLPage(props: Props) {
             </div>
           </Container>
         )}
-        <LoginModal
-          name='Login'
-          open={isModalOpen}
+        {isModalOpen ? <LoginModal
+          name='WorldACLPageLoginModal'
           onClose={() => setIsModalOpen(false)}
           onConnect={onConnectWallet}
           isLoading={isConnecting}
-        />
+        /> : null}
```

Comment:
This change and the other similar ones are because the `open` property even tho still exist, it doesn't work anymore, because there was a change on `decentraland-dapps` to support controlling this from redux and since then the property is not usable, so I changed the syntax to a ternary expression.